bothub/api/v1/serializers/example.py

```diff
@@ -168,6 +168,8 @@ def create(self, validated_data):
             language = None
         repository_update = repository.current_update(language or None)
         validated_data.update({'repository_update': repository_update})
+        repository.total_updates += 1
```

Comment:
Não é responsabilidade do serializer fazer este tipo de operação e sim da model, se fizer desse modo se em algum outro local for chamado o "teste" ele nunca vai contabilizar pois isso ficou no serializer... outra coisa isso tem que ser executado no NLP ao fim do teste com succeso para garantir a contagem correta, senão será incrementado o teste no engine mas no NLP ele falhou, fica informação errada.